test/ci/launch.yml

```diff
@@ -0,0 +1,50 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: list available AMIs
+      ec2_ami_find:
+        region: '{{ aws_region }}'
+        ami_tags:
+          operating_system: '{{ operating_system }}'
```

Comment:
Let's namespace these types of variables.

Reply:
Also, we can parameterize the ami_tags into a dictionary; This will allow us to provide whatever we want to finding an AMI if we tag things differently later.

Reply:
Removed - now `aws_ami_tags` can be host-specific